scripts/lint-lua-ls.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# lua-language-server should be installed automatically by the flake.nix dev shell
+# or .github/workflows/pre-commit.yml
+if ! type lua-language-server &>/dev/null; then
+  echo "ERROR: lua-language-server is not installed. Please run 'nix develop' or install it manually."
+  exit 1
+fi
+
+if [ ! -e "${CURSORLESS_REPO_ROOT-nonexistent}" ]; then
+  CURSORLESS_REPO_ROOT=$(git rev-parse --show-toplevel)
+fi
```

Comment:
@AndreasArvidsson this is what we should use for that script you removed when we reintroduce it. Not sure why we didn't think of this